src/app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\LoginRequest;
+use Illuminate\Support\Facades\Auth;
+
+class LoginController extends Controller
+{
+
+    public function __invoke(LoginRequest $request)
+    {
+        $validated = $request->validated();
+
+        $credentials = $request->only('username', 'password');
+
+        if (Auth::attempt(['name' => $credentials['username'], 'password' => $credentials['password']])) {
```

Comment:
Remove $creadentials line, and instead of use $credentials[...] to login use $validated